src/cms/templates/events/event_list.html

```diff
@@ -27,23 +41,30 @@ <h2 class="heading font-normal mb-2">{% trans 'Events' %}</h2>
         <div class="flex flex-wrap">
             <form class="table-search relative w-1/2">
                 <i data-feather="search" class="absolute"></i>
-                <input type="search" placeholder="{% trans Search %}" class="h-full pl-10 pr-4 rounded shadow">
+                <input type="search" placeholder="{% trans 'Search' %}" class="h-full pl-10 pr-4 rounded shadow">
             </form>
-            {% if region.default_language == language %}
             <div class="w-1/2 flex flex-wrap justify-end">
-                <a href="{% url 'new_event' region_slug=region.slug language_code=language.code %}"
-                   class="bg-blue-500 hover:bg-blue-600 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline">
-                    {% trans 'Create event' %}
-                </a>
+                <button id="filter-toggle"
+                        class="bg-red-500 hover:bg-red-600 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline mr-3">
```

Comment:
```suggestion
                        class="bg-gray-500 hover:bg-gray-600 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline mr-3">
```
Same thing here, I think a red button should be used for "dangerous" buttons (e.g. delete), not for an innocent filter button :wink:

Reply:
Done!